src/rust/e2e-tests/tests/sysmon_log_e2e_test.rs

```diff
@@ -41,40 +51,55 @@ async fn test_sysmon_log_e2e(ctx: &mut E2eTestContext) -> Result<(), Box<dyn std
 
     tracing::info!(">> Setup complete. Now let's test milestones in the pipeline.");
 
-    let raw_logs_scanner = KafkaTopicScanner::new(ConsumerConfig::with_topic("raw-logs"))?
-        .contains_for_tenant(tenant_id, |_log: RawLog| true)
-        .await?;
-
-    let generated_graphs_scanner =
-        KafkaTopicScanner::new(ConsumerConfig::with_topic("generated-graphs"))?
-            .contains_for_tenant(tenant_id, |graph: GraphDescription| graph.nodes.len() > 1)
-            .await?;
-
-    let node_identifier_scanner =
-        KafkaTopicScanner::new(ConsumerConfig::with_topic("identified-graphs"))?
-            .contains_for_tenant(tenant_id, events_36lines_node_identity_predicate)
-            .await?;
-
-    let graph_merger_scanner = KafkaTopicScanner::new(ConsumerConfig::with_topic("merged-graphs"))?
-        .contains_for_tenant(tenant_id, events_36lines_merged_graph_predicate)
-        .await?;
+    let raw_logs_scanner_handle = KafkaTopicScanner::new(
+        ConsumerConfig::with_topic("raw-logs"),
+        Duration::from_secs(30),
+    )
+    .contains_for_tenant(tenant_id, 36, |_log: RawLog| true)
+    .await;
+
+    let generated_graphs_scanner_handle = KafkaTopicScanner::new(
+        ConsumerConfig::with_topic("generated-graphs"),
+        Duration::from_secs(30),
+    )
+    .contains_for_tenant(tenant_id, 36, |_graph: GraphDescription| true)
+    .await;
+
+    let node_identifier_scanner_handle = KafkaTopicScanner::new(
+        ConsumerConfig::with_topic("identified-graphs"),
+        Duration::from_secs(30),
+    )
+    .contains_for_tenant(tenant_id, 36, |_graph: IdentifiedGraph| true)
```

Comment:
can we bring back my predicates?

Reply:
oh i see they're done below in-line, which i'm happy about; but i'm a bit confused about the role of the predicate here now if we do filtering later on

Reply:
Yeah, the idea is to just do _filtering_ in the `contains`, then make assertions on the filtered results. It just makes it a lot easier to tell what went wrong when tests fail.

I think maybe I should change the `contains` names to be `filter`, actually. Does that sound OK?

Reply:
done. Changed them to `scan` and `scan_for_tenant`